docs/source/models/templates/concept_model_template.rst

```diff
@@ -47,19 +47,14 @@
 
 
 
-.. _concept_model_template:
-
-.. todo::
-
-  Replace the above ".. _concept_model_template:" with ".. _{YOUR_MODEL_NAME}:""
```

Comment:
I think these lines should remain in place for the template because the `.. _concept_model_template:` target allows other documentation pages to link to the template page specifically. When individual concept model pages are created, they should each get their own label of the form `.. _{YOUR_MODEL_TITLE}:`.